superlesson/steps/annotate.py

```diff
@@ -36,6 +36,46 @@ def to_pdf(self):
             merger.write(f)
         logging.info(f"Annotated PDF saved as {output}")
 
+    @staticmethod
+    def emphasize(text: str) -> str:
+        return "_" + text + "_"
+
+    @classmethod
+    def fade_slide(cls, current: str, next: str, threshold: int = 20) -> tuple[str, str]:
+        dots = ['.']
```

Comment:
As discussed yesterday we should be checking for any punctuation here:

```suggestion
        dots = ['.?!']
```